CMakeLists.txt

```diff
@@ -64,6 +64,10 @@ set( CMAKE_CXX_EXTENSIONS OFF )
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 
+if( T8CODE_BUILD_FORTRAN_INTERFACE )
+    enable_language( Fortran )
```

Comment:
Why has this been moved out of line 155 into its own block?

Reply:
Because a few lines below there is the following:
```
if( T8CODE_BUILD_FORTRAN_INTERFACE )
      find_package( MPI COMPONENTS C Fortran REQUIRED )
[...]
```
When `Fortran` is not enabled beforehand CMake throws an error.